sdks/java/core/src/test/java/org/apache/beam/sdk/transforms/ParDoTest.java

```diff
@@ -3311,6 +3310,47 @@ public void processElement(
     }
   }
 
+  /** Tests validating SideInput annotation behaviors. */
+  @RunWith(JUnit4.class)
```

Comment:
Don't think we need a new test class here. A new test method (right after the existing testParDoWithSideInputs method) should be fine.

Reply:
done